Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -108,9 +95,28 @@ public void UpdateImages(int objectId, List<ImageDto> imageDtos)
         }
 
         context.SaveChanges();
+    }
+
+    public void UpdateImage(int objectId, ImageDto imageDto)

```

Comment:
Since it's designed like it's only working for User, can we call it UpdateImageForUser then? and get int userId, just to have it clear as we can't edit specific image, only first of object, so it would only work correctly for user

Reply:
changed